x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/rollup/job/RollupIndexer.java

```diff
@@ -115,6 +126,12 @@ protected void onStart(long now, ActionListener<Boolean> listener) {
         }
     }
 
+    @Override
+    protected void onFinish(ActionListener<Void> listener) {
+        final RefreshRequest refreshRequest = new RefreshRequest(job.getConfig().getRollupIndex());
+        client.admin().indices().refresh(refreshRequest).actionGet();
```

Comment:
What happens if the call to `refresh` fails? `actionGet()` may throw exceptions here?

Also, I think the call to `refresh` sould be async by calling method

```java
client.admin().indices().refresh(RefreshRequest request, ActionListener<RefreshResponse> listener)
```

The big question is what happens if the call to `refresh` fails. 
Since we got there, the index has been created successfully and we should not fail the whole job.

Reply:
I guess in both cases, successful or not we can just log something.